src/inflators/audio.js

```diff
@@ -0,0 +1,24 @@
+import { absoluteURLForAsset, addComponent } from "./utils";
+
+export function inflateAudio(node, componentProps, otherComponents) {
+  // In some cases in Hubs GLBs, the audio-params component is not present,
+  // and the audio component has the audio parameters directly.
+  // In audio-params component, volume is called gain.
+  const hasAudioParams = typeof otherComponents["audio-params"] !== "undefined";
+  const audioParams = otherComponents["audio-params"] || componentProps;
+
+  // Set properties.
+  const audioProps = {
+    autoplay: componentProps.autoPlay,
+    distanceModel: audioParams.distanceModel,
+    loop: componentProps.loop,
+    maxDistance: audioParams.maxDistance,
+    positional: audioParams.audioType !== "stereo",
+    refDistance: audioParams.refDistance,
+    rolloffFactor: audioParams.rolloffFactor,
+    src: absoluteURLForAsset(componentProps.src),
+    volume: hasAudioParams ? audioParams.gain : audioParams.volume * 3,
```

Comment:
Multiplying by 3 is what hubs code does?

Reply:
No, but without the multiply the sound is so low, that you can hardly hear it. I didn't find what values are used for the alternative audio component, so I tried to get about the same volume I hear when the GLB is loaded in Hubs.

Reply:
Ok fair we don't have the same exact sound system/component as hubs here, so handling the volume may be different. And if we have an audio-params component, there is not need to multiply by 3?